### DISCUSSION TITLE
Icaros 3.3.3 Beta 2

### DISCUSSION THREAD

**Discussion by Xanashi** [Xanashi/Icaros#107] [Releases]
Good Afternoon Everyone,

I have another release today, adding thumbnailing support for EPS and INDD files, 
and a fix for the thumbnailing of 'tiled' HEIC files, that was reported with the previous Beta.

Be sure to report any bugs or regressions as always.

I hope you all enjoy the new release and have a wonderful summer! ❤️ 

<br/>

[![Icaros releases](https://img.shields.io/github/downloads/xanashi/icaros/v3.3.3b2/total?color=%232B6870&label=DOWNLOADS&style=for-the-badge)](https://github.com/Xanashi/Icaros/releases/tag/v3.3.3b2)

_______________________________________________________________________________________________________

### Icaros v.3.3.3 Beta 2

- Added support for thumbnailing 'tiled' HEIC files (Fixes 'zoomed in' thumbnails for HEIC files)
- Added thumbnailing support of EPS files with an embedded preview
- Added thumbnailing support of INDD files with an embedded preview
- Added AI, EPS, and INDD to the 'Common' and 'Most Known' presets
- Improved thumbnailing of AI files
- AI, EPS and INDD are now registered as image formats
- The 'View Size' will now only show up Windows 7 and below, as it is not relevant on newer versions of Windows
- A couple of minor improvements
- Updated FFmpeg

<hr /><em>This discussion was created from the release <a href='https://github.com/Xanashi/Icaros/releases/tag/v3.3.3b2'>Icaros 3.3.3 Beta 2</a>.</em>

**Comment by Klaus1189** [comment]
Thank you so much ❤️

**Reply by Xanashi** [reply]
You're welcome! ❤️

**Comment by Qhqmvga7BWJs39sHZ** [comment]
InDesign thumbnails are showing up as corrupted for me.  I have tried with a few different files.  See screen captures.
![Screenshot 2024-07-22 124601](https://github.com/user-attachments/assets/44b31225-503e-4a36-817d-da5405f241a5)
![Screenshot 2024-07-22 124337](https://github.com/user-attachments/assets/c7fd527e-cbbb-46c1-8abd-30d6b14f52be)

**Reply by Xanashi** [reply]
Thank you for reporting. 
Would it be possible to share these files with me? It would help a lot with debugging the issue.

**Reply by Qhqmvga7BWJs39sHZ** [reply]
Here's one file.  Sometimes the thumbnail works for other files.
[TEST.zip](https://github.com/user-attachments/files/16354808/TEST.zip)

**Reply by Xanashi** [reply]
Thank you for the sample!
This should be fixed in [3.3.3 Beta 3](https://github.com/Xanashi/Icaros/releases/tag/v3.3.3b3).
